src/OpenTelemetry.Abstractions/Trace/ISpanBuilder.cs

```diff
@@ -109,6 +110,13 @@ public interface ISpanBuilder
         /// <returns>This span builder for chaining.</returns>
         ISpanBuilder SetRecordEvents(bool recordEvents);
 
+        /// <summary>
+        /// Set start UTC timestamp value.
+        /// </summary>
+        /// <param name="startTimestamp">Start timestamp in UTC. When not set, precise current UTC time is used.</param>
```

Comment:
> When not set

It cannot be not set. Do you mean When `DateTime.MinValue`? Or you think we should accept a nullable type? I'd say remove this comment so customers will simply not call this method.

Unless this is what you meant by "when not set" =). Than I' rephare "when this method wasn't called, the precise current time at the moment of StartSpan call will be used"

Reply:
I mean `default(DateTime)`